example_dags/example_append.py

```diff
@@ -0,0 +1,48 @@
+import pathlib
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+
+from astro import sql as aql
+from astro.files import File
+from astro.sql.table import Table
+
+CWD = pathlib.Path(__file__).parent
+
+default_args = {
+    "owner": "airflow",
+    "retries": 1,
+    "retry_delay": 0,
+}
+
+dag = DAG(
+    dag_id="example_append",
+    start_date=datetime(2019, 1, 1),
+    max_active_runs=3,
+    schedule_interval=timedelta(minutes=30),
+    default_args=default_args,
+)
+
+with dag:
+    load_main = aql.load_file(
+        input_file=File(path=str(CWD) + "/data/homes.csv"),
```

Comment:
We could also create a DATA_DIR as a variable with `DATA_DIR = CWD + "/data/"` and reuse it here and in the below `load_append` task

Reply:
Done.